examples/ltsf/ltsf_example.py

```diff
@@ -0,0 +1,48 @@
+"""Test LTSF."""
+import random
+
+import pandas as pd
+import torch
+
+from sktime.forecasting.ltsf import LTSFLinearForecaster
+
+
+def train_test_split(dataset_path, input_output_len, test_size):
+    """Shuffle data for LTSF-Linear."""
+    data = pd.read_csv(dataset_path).drop("date", axis=1)
+    data = [
+        data[i : i + input_output_len] for i in range(0, len(data), input_output_len)
+    ]
+    data = [x for x in data]
+    random.shuffle(data)
```

Comment:
Hey, I am here from sktime discord (lkihn) and I was wondering whether this train-test separation was done the same way in paper or is this draft part that is to be rewritten.

I am asking as this can lead to misleading results due to overfitting compared to time based split.
Thanks!

Reply:
I think this is probably just a draft to test functionality - a temporal split is done in 19/20
but @luca-miniati would know best

Reply:
I found everything I needed here: https://github.com/cure-lab/LTSF-Linear/blob/main/data_provider/data_loader.py#L235-L241

Thanks.